addons/point_of_sale/static/src/app/navbar/sync_notification/sync_notification.xml

```diff
@@ -15,10 +15,7 @@
                 <i class="fa fa-fw fa-spin fa-circle-o-notch" role="img"
                    aria-label="Synchronisation Connecting" title="Synchronisation Connecting"></i>
             </div>
-            <div t-else="" class="js_connected oe_icon text-success">
-               <i class="fa fa-fw fa-wifi" role="img" aria-label="Synchronisation Connected"
-                   title="Synchronisation Connected"></i>
-            </div>
+            <div t-else="" class="js_connected oe_icon text-success" style="visibility: hidden;"/>
```

Comment:
can't you remove this line ?

Reply:
It's needed for this check https://github.com/odoo/odoo/blob/dee10df5c180ef376f4d334e18e4ee2de562ffe6/addons/pos_restaurant/static/tests/tours/pos_restaurant_tour.js#L16-L23

Reply:
before, the way to see that you are connected was to look for the green icon; now, as long as you don't see the "disconnected" icon, it means that you are connected, so you can change the tours to check that you don't see the "disconnected" icon instead

Reply:
before, the way to see that you are connected was to look for the green icon; now, as long as you don't see the "disconnected" icon, it means that you are connected, so you can change the tours to check that you don't see the "disconnected" icon instead